lib/src/utils/scope.dart

```diff
@@ -0,0 +1,20 @@
+abstract class Scope {
+  const Scope(this.permission);
+
+  final String permission;
+
+  @override
+  String toString() => permission;
+}
+
+class EmailAddressScope extends Scope {
+  const EmailAddressScope() : super('r_emailaddress');
```

Comment:
Please add simple tests for this file, checking provided const value (toString method)

Reply:
sure